### DISCUSSION TITLE
What's your number one feature missing from Pocket?

### DISCUSSION THREAD

**Discussion by andyw8** [open-pocket/open-pocket#4] [General]
What features do you wish Pocket had?

(see also: https://help.getpocket.com/article/1063-pockets-top-feature-requests )

**Comment by KevinMThurman** [comment]
The number one feature missing from Pocket is the lack of integration. Examples:

Integration with other reading options: Newsletters that come into my Gmail and kindle books are completely separate. Yet, I highlight things in each one and need a separate app to 

Integration with other apps: Given the above, I have tried Roam, Mem, Notion, and others to keep notes together, but zapier integration is not customizable enough to ensure that they are integrated. 

For this reason, I have found myself looking for a solution other than Pocket for these and other use cases.

**Comment by ghost** [comment]
@KevinMThurman @andyw8 Hi! How are you?

1. There is this extension [SingleFile](https://github.com/gildas-lormeau/SingleFile) that I use to save my sites locally, I wonder if this could be an open source alternative to pocket openly and in client
2. I don't know if you all know more there is a technical discussion of implementing something similar to Pocket in the browser, I think the idea is interesting and as you all are talking about creating an open source pocket solution it is worth taking a look at this discussion here [integrating-websites-with-the-browser](https://discourse.wicg.io/t/integrating-websites-with-the-browser/2512) and here [Single Page maybe solve this WIGC problem](https://github.com/gildas-lormeau/SingleFile/issues/997)

**Comment by garibarba** [comment]
Twitter thread unrolling, in the style of https://threadreaderapp.com/

**Comment by tsujigiri** [comment]
For me, it's adding context to saves. Where did I find the link? Notes. There is a feature where Pocket saves the Tweet you got a link from, if you save it from the Twitter app. I created a [Firefox add-on](https://addons.mozilla.org/de/firefox/addon/pockettoots/), which sort of does this for Mastodon posts. But I would like to get more context, collected automatically wherever possible.

**Comment by thomrommens** [comment]
I would like to have an extra parameter to the `/add` API, for indicating cookies are to be accepted at the targeted URL. 
So that the Pocket internal engine handles the cookie in the HTTP request.
Now, I often get a link to a cookie wall in my Pocket saved articles instead.
